ChangeLog.txt

```diff
@@ -72,6 +74,9 @@ BUGFIXES:
 - Fixed the oversight which made it impossible to keep the Bureaucratic government as a Romance empire after adopting it
 - Fixed Zoroastrian vassals of Persia being feudal government rather than Eransharhr
 - Fixed the Bubonic plague being announced in events without it being on the map
+- Fixed Western Roman Imperial Regalia artifact spawns
```

Comment:
How did you fix them? This entry could be more precise in my opinion.

Reply:
Done, described it more precisely.